solr/core/src/test/org/apache/solr/cloud/TestConfigSetsAPI.java

```diff
@@ -293,8 +292,7 @@ private NamedList getConfigSetPropertiesFromZk(
   private void verifyProperties(String configSetName, Map<String, String> oldProps,
        Map<String, String> newProps, SolrZkClient zkClient) throws Exception {
     @SuppressWarnings({"rawtypes"})
-    NamedList properties = getConfigSetPropertiesFromZk(zkClient,
-        ZkConfigManager.CONFIGS_ZKNODE + "/" + configSetName + "/" + DEFAULT_FILENAME);
+    NamedList properties = getConfigSetPropertiesFromZk(zkClient,ZkConfigSetService.CONFIGS_ZKNODE + "/" + configSetName + "/" + DEFAULT_FILENAME);
```

Comment:
`getConfigSetPropertiesFromZk` circumvents the ConfigSetService; could it use it?